lib/dams/controller_helper.rb

```diff
@@ -882,5 +882,15 @@ def referrer_controller( request )
         end
       end
     end
+
+  def metadata_display?(data)
+    result = false
+    unless data.nil?
```

Comment:
I think this could be `if data` or you could drop the conditional and do a `return unless data`